Source/Engine/Physics/Physics.h

```diff
@@ -573,5 +589,9 @@ DECLARE_SCRIPTING_TYPE_NO_SPAWN(Physics);
     /// Marks that joint has been removed (all collision events should be cleared to prevent leaks of using removed object).
     /// </summary>
     /// <param name="joint">The joint.</param>
-    static void RemoveJoint(Joint* joint);
+    static void RemoveJointAll(Joint* joint);
```

Comment:
I think we could stick to the previous name.

Reply:
Done